group_vars/all

```diff
@@ -2,7 +2,7 @@ geekbench: "Geekbench-3.4.1-Linux"
 
 geekbench_stdout: "{{ lookup('file', 'test_geekbench_op.txt') }}"
 
-benchmarking_dir: "/root/benchmarking"
+benchmarking_dir: "/home/abhinav/benchmarking-playbook/benchmarking"
```

Comment:
This should perhaps go into a group_vars/local file as it is specific to one machine.